examples/CosmosDB/CosmosDB.Tests.ps1

```diff
@@ -177,7 +179,65 @@ Describe 'Verify Cosmos DB Account' {
       PropertyValue     = $sqlDatabaseName
     }
 
-    # act and assert
+    #act
+    Confirm-AzBPResource @params | Should -BeSuccessful
+  }
+
+  It "Should contain a Cosmos DB SQL Role Assignment named $sqlRoleAssignmentId - Confirm-AzBPResource" {
+    #arrange
+    $params = @{
+      ResourceType      = "CosmosDBSqlRoleAssignment"
+      ResourceGroupName = $rgName
+      AccountName       = $sqlAccountName
+      RoleAssignmentId  = $sqlRoleAssignmentId
+    }
+
+    #act
+    Confirm-AzBPResource @params | Should -BeSuccessful
+  }
+
+  It "Should contain a Cosmos DB SQL Role Assignment named $sqlRoleAssignmentId - Confirm-AzBPResource" {
+    #arrange
+    #TODO: check if this is the correct property
```

Comment:
Oh just noticed this TODO and the one on line 230. Please remove these comments if the TODOs are completed.